rosbag2_converter_default_plugins/CMakeLists.txt

```diff
@@ -24,55 +24,57 @@ find_package(rosbag2 REQUIRED)
 find_package(rosidl_generator_cpp REQUIRED)
 find_package(rcutils REQUIRED)
 find_package(rmw REQUIRED)
+find_package(rmw_fastrtps_cpp QUIET)
 
-add_library(${PROJECT_NAME} SHARED
-  src/rosbag2_converter_default_plugins/cdr/cdr_converter.cpp)
+if(rmw_fastrtps_cpp_FOUND)
```

Comment:
I would somehow inverse this logic and return `if(NOT rmw_fastrtps_cpp_FOUND)` with at least a notification that this library is not going to be compiled in full. This will also ease the diff a little bit as we don't have an extra layer of indentation.